test/integration/test_grpc_service.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 import pytest
 
-from test.integration.integration_test_fixtures import http_test_server_fixture
+from test.integration.integration_test_fixtures import (http_test_server_fixture, server_config)
```

Comment:
(optional for this PR) comment about only importing modules / packages applies here too.

Optional, because it is pre-existing and we probably don't want to increase the scope of this PR.